service/service.go

```diff
@@ -62,5 +62,8 @@ func (s *CaptionsService) Endpoints() map[string]map[string]http.HandlerFunc {
 		"/jobs/{id}/cancel": {
 			"POST": server.JSONToHTTP(s.CancelJob).ServeHTTP,
 		},
+		"/jobs/{id}/download": {
+			"POST": server.JSONToHTTP(s.DownloadCaption).ServeHTTP,
```

Comment:
I think this shouldn't be a JSON endpoint. I guess we want to return just the contents of the captions, but when we do ``server.JSONToHTTP``, Gizmo will marshal the interface returned by the handler as JSON, which would probably fail at this point.

That would require changing the signature of DownloadCaption to ``func(r *http.Request, w http.ResponseWriter)`` and just register it as:

```
"POST": s.DownloadCaption,
```

And we should also make sure that ``DownloadCaption`` sets the proper ``Content-Type`` when returning the content (if we use ``JSONToHTTP``, the ``Content-Type`` will be ``application/json``).